augur/application/config.py

```diff
@@ -66,7 +66,9 @@ def get_development_flag():
                 "log_level": "INFO",
             },
             "Celery": {
-                "concurrency": 12
+                "concurrency": 12,
+                "worker_process_vmem_cap": 0.25,
+                "refresh_materialized_views_interval_in_days": 7
             },
```

Comment:
I don't think we still need the `concurrency` config option, since we have `worker_process_vmem_cap` now

Reply:
You're right I didn't notice that